lib/nexpose/scheduled_backup.rb

```diff
@@ -0,0 +1,79 @@
+module Nexpose
+# Constants useful across the Nexpose module.
+# Configuration structure for scheduled backups.
+  class ScheduledBackup < APIObject
+    require 'json'
+    include JsonSerializer
+
+    # Whether or not this schedule is enabled.
+    attr_accessor :enabled
+    # Valid schedule types: daily, hourly, monthly-date, monthly-day, weekly.
+    attr_accessor :schedule_type
+    # The repeat interval based upon type.
+    attr_accessor :schedule_interval
+    # The earliest date to generate the report on (in ISO 8601 format).
+    attr_accessor :schedule_start
+    # The description of the backup
+    attr_accessor :description
+    # Whether the backup will be platform independent or not
+    attr_accessor :platform_independent
+    # Whether the backup should pause all local scans or wait for local scans to complete.
+    attr_accessor :pause_local_scans
+    # Cancellation window in minutes for scheduled DB maintenance.
+    attr_accessor :cancellation_period
+
+    def initialize(start, enabled=true, type, interval, platform_independent, description, pause_local_scans, cancellation_period)
```

Comment:
Also be sure to update the attr docs to indicate what defaults are set if you do this.